test_themes/static/src/systray_items/website_switcher.js

```diff
@@ -12,6 +12,8 @@ patch(WebsiteSwitcherSystray.prototype, 'test_themes_website_switcher_systray',
 
         this.orm = useService('orm');
         this.tooltips = useState({});
+        // Disable the notification service to avoid having a notification for each theme.
+        this.notificationService = { add: () => () => null };
```

Comment:
It means that should we add another notification in the `WebsiteSwitcherSystray` component (and that we would not want to disable it with this module), it won't work with this module.
It's really not a big deal I guess, it really just is a test module.
Hopefully we don't get confuse later if we add another notification there and don't know why it's not working on the design-themes runbot, but then again, it's overthinking.

I think I would have just done that in the `getElements()` and not the `setup()` if it is even possible so we only alter the behavior in this method and not the whole component (and of course restore the original notificationService at the end of the method).

Reply:
I thought about that possibility too.

If we wanted to avoid this behaviour with a bit more dev, I would have just moved the notification part into its own method so that I could have overiden it with the patch. Because I think if we only disable it during getElements, the callbacks would be called after the call to that method, and therefore trigger a notification anyway.

Hopefully if someone realises it doesn't show on design-themes, they can quickly find this bit of code + the commit that introduces it and understand what is being done.